### PR TITLE
Fixed styles (Addresses #267)

### PR DESCRIPTION
**Pull request by siddhant1**
ref #267

### CODE REVIEW

**Comment by siddhant1** [comment]
image

@algomaster99 would you like to see any more changes ?

**Comment by siddhant1** [comment]
@domoritz I think we should change the color of the link or do something like this to let people know that this is a link

image

**Comment by siddhant1** [comment]
I have changes ready , tell me if you want this .

**Comment by algomaster99** [comment]
@siddhant1 
- Margin between heading anf buttons can be more consistent
- Font-size of headers should be uniform
These are my suggestions. If you want to incorporate more, you may :)

**Comment by siddhant1** [comment]
Hm , I think I should change them
Thanks for suggestions :)

**Comment by siddhant1** [comment]
Final Styles , also fixed link to docs styles
image

image

**Comment by domoritz** [comment]
I'm not a huge fan of the links as buttons. Can you instead just make them blue with an underline?

File: src/components/help-modal/index.css [hunk 1]
```diff
@@ -65,4 +73,20 @@ kbd {
 .doc-image {
   height: 20px;
   margin: 0px 6px 0px 2px;
+  vertical-align: sub;
+}
+
+.tips {
+  text-decoration: none;
+  color: black;
```
Comment (domoritz) [inline src/components/help-modal/index.css]: Why override it?

Comment (siddhant1) [reply src/components/help-modal/index.css]: Was overriding to make the link a button , removed

**Comment by siddhant1** [comment]
image
Looks like this now

**Comment by siddhant1** [comment]
Removed redundant margin 

Looks like this now 
image

**Comment by domoritz** [comment]
The alignment is still terrible. Also, could you make the follow and reference links actual links instead of buttons?

**Comment by siddhant1** [comment]
Sure

**Comment by siddhant1** [comment]
Do you want something similar to https://github.com/vega/editor/pull/260#issuecomment-471585781

**Comment by domoritz** [comment]
Yes

**Comment by siddhant1** [comment]
image

**Pull request merged**